src/mame/drivers/at.cpp

```diff
@@ -2588,12 +2711,20 @@ COMP( 1994, ficvipio,  ibm5170, 0,       at486,     0,     at_state,     init_at
 COMP( 199?, ficvipio2, ibm5170, 0,       at486,     0,     at_state,     init_at,        "First International Computer", "486-VIP-IO2", MACHINE_NOT_WORKING )
 COMP( 199?, hot409,    ibm5170, 0,       at486,     0,     at_state,     init_at,        "Shuttle Computer International", "HOT-409", MACHINE_NOT_WORKING )
 COMP( 198?, aubam12s2, ibm5170, 0,       atvga,     0,     at_state,     init_at,        "AUVA COMPUTER, INC.", "BAM/12-S2", MACHINE_NOT_WORKING )
+COMP( 198?, wyse286,   ibm5170, 0,       ibm5162,   0,     at_state,     init_at,        "Wyse", "WYSEpc", MACHINE_NOT_WORKING )
```

Comment:
This machine should have a more specific full name. The first WYSEpc (WY-1100) was a mostly-compatible XT clone.

Reply:
Right, the WYSEpc one is no problem, I'll give it a better name.
Now how about CS8220 stuff ... do you want them in the "at" section or the "atvga" section?

How about those doublettes between at and atvga. Vas argued that they should be kept as they would be faster 286 machines with a VGA card as standard. Shouldn't they then be deleted from the "at" section?